test/zebras/test_period.py

```diff
@@ -114,4 +114,3 @@ def test_periods_feature(freq):
 
     np.testing.assert_array_equal(pr.start_time.dayofweek, ps.dayofweek)
     np.testing.assert_array_equal(pr.start_time.dayofyear, ps.dayofyear)
-    np.testing.assert_array_equal(pr.start_time.week, ps.week)
```

Comment:
@jaheba looks like `week` is just not in `DatetimeIndex` anymore (can't blame them since weeks can span across two years), so I'm just removing this.